src/test/EigenLayerDeployer.t.sol

```diff
@@ -101,7 +99,7 @@ contract EigenLayerDeployer is Operators {
     address podAddress;
     address delayedWithdrawalRouterAddress;
     address eigenPodBeaconAddress;
-    address beaconChainOracleAddress;
+    address beaconChainOracleAddress = 0x40B10ddD29a2cfF33DBC420AE5bbDa0649049f2c;
```

Comment:
seems odd to be hard-coding this address in the tests? I guess we just aren't using it so it's OK?

Reply:
What do you think is best here? We removed the Oracle contract that wasn't being used in the tests, but we still needed to pass it as a parameter to some contracts. It could have been an uninitialized value, but that similarly felt weird.

Reply:
ah, good point. I think it's OK as-is since we aren't using it.
Going forward we should probably make some mock for this and use it in tests as needed.

Reply:
That’s a good point. I will make a mock so it’s clearer

Reply:
addressed in [935a629](https://github.com/Layr-Labs/eigenlayer-contracts/pull/177/commits/935a629e2814ce8ab8c4f8897839861116358e54)